src/wallets/TransactionRow.scss

```diff
@@ -134,7 +134,7 @@
 
     &.incoming {
       @include themify($themes) {
-        color: themed('success-color');
+        color: $primary-color;
```

Comment:
`themify` is usually not needed if you don't use a `themed` function inside.
You can use it for the side-effect though, to wrap the rule in e.g. `.theme-dark` (can help with specificity issues).

Nonetheless, normally I wouldn't use it for the side effect and I'd specify a new theme variable where the color is the same for both dark and light theme. If we introduce a new theme where `$primary-color` doesn't work with the bg, we'd have to hunt down these "hardcoded" colors.

Reply:
Yes, I forgot to remove `themify` here. I don't think this value should be themified, because the background here is already a static color. So either both of those should be themified (and in this case same for both themes) or both can be static.